scripts/createReviewAppNamespace.js

```diff
@@ -0,0 +1,43 @@
+const CLOUDFLARE_API_URL = "https://api.cloudflare.com/client/v4"
+const CLOUDFLARE_ZONE_ID = "38e46d6be412f00b9fc0ef478f497ef1"
+const CLOUDFLARE_API_TOKEN = process.env.CLOUDFLARE_API_TOKEN
+const BRANCH_NAME = process.env.CIRCLE_BRANCH
+
+async function addCnameRecord() {
+  const dnsSubdomain = BRANCH_NAME.replace("review-app-", "")
+
+  const response = await fetch(
+    `${CLOUDFLARE_API_URL}/zones/${CLOUDFLARE_ZONE_ID}/dns_records`,
+    {
+      method: "POST",
```

Comment:
Is this idempotent (as in, pushing subsequent commits to the review app branch)?

Reply:
Whoops sorry for double-commenting about this.

Reply:
It outputs a helpful error: https://app.circleci.com/pipelines/github/artsy/force/66840/workflows/37d18afe-72e6-4210-aa4f-b8f162047242/jobs/482747

I like this better than performing two fetches. Its descriptive, gets the job done and doesn't fail.